### ISSUE TITLE
jsdom.jsdom: Getting included scripts to execute.

### ISSUE THREAD

**Issue by Michael-F-Ellis** [tmpvar/jsdom#1761]
Hi, First time user of jsdom here.  I've got the following script cribbed together from the README and various answers on stack overflow.  It runs without error when I supply it an html file and gives me back the file contents in the output but doesn't execute the js scripts included in the file that modify the html.  

What must I add to get that to happen?  

Thanks!
```
var jsdom = require("jsdom").jsdom;
var serializeDocument = require("jsdom").serializeDocument;
var fs = require('fs');

if (process.argv.length < 3) {
  console.log('Usage: node ' + process.argv[1] + ' FILENAME');
  process.exit(1);
}

var file = process.argv[2];
var outfile = "jsdomtest-output.html"
fs.readFile(file, 'utf8', function(err, data) {
    if (err) throw err;
    var doc = jsdom(data);
    var content = serializeDocument(doc);
    fs.writeFile(outfile, content, function(err) {
       if (err) throw err;
       console.log('It\'s saved!');
    });
}); 
```

**Comment by domenic** [comment]
If you are using `jsdom.jsdom`, you need to use FetchExternalResources/ProcessExternalResources.

BTW, we are aware this API is terrible and unintuitive. It is being fixed in the next major version. See #1741, and in particular I'd love your thoughts on the new draft README at https://github.com/tmpvar/jsdom/blob/newapi/README.md

**Comment by Michael-F-Ellis** [comment]
Thanks for the quick response!  I just skimmed the new README and it certainly seems cleaner, but truthfully I don't have enough experience with the old version (or node.js for that matter to make a really informed comment.  I will take a second look after I get the code I've got working.  

When you say 'use the FetchExternalResources/ProcessExternalResources' does that mean I attach them wrapped in an object as a second argument to my jsdom(data) call? And do I need to have an onload binding that invokes the serializeDocument() and fs.writeFile() calls or can I leave those as I now have them?

Thanks again,
Mike
